.github/pull_request_template.md

```diff
@@ -16,8 +16,14 @@
 ### How Has This Been Tested?
 
 Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.
+Include log analysis if applicable.
 
 1. Test 1
 - Step 1
 - Step 2
 - Step 3
+
+Each PR (into master) that touches a live codepath (as opposed to tests or local tooling) requires at least one unit test, so we can slowly work our way to better coverage and make all our lives easier :)
```

Comment:
I feel like this phrasing feels a bit unecessary. My take is we just say like "Adding tests is encouraged. Please list any tests you added:"